openssl-sys/src/bio.rs

```diff
@@ -34,37 +34,37 @@ pub unsafe fn BIO_get_mem_data(b: *mut BIO, pp: *mut *mut c_char) -> c_long {
 
 extern "C" {
     #[deprecated(note = "use BIO_meth_set_write__fixed_rust instead")]
-    #[cfg(any(ossl110, libressl273))]
+    #[cfg(ossl110)]
```

Comment:
Is there someting enabling this for libressl via ossl110? Otherwise this doesn't look right.